apps/dashboard/app/javascript/apps.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 
+import { configData } from './config.js';
```

Comment:
Can you add a real function to `config.js`. I just don't like pulling the config data by itself and all these disparate files having to know the keys for it.

Reply:
Fixed